masonry/src/theme.rs

```diff
@@ -29,6 +29,10 @@ pub const ACCENT_COLOR: Color = Color::from_rgb8(0x3b, 0x7e, 0xe4);
 pub const TEXT_COLOR: Color = Color::from_rgb8(0xf0, 0xf0, 0xea);
 pub const DISABLED_TEXT_COLOR: Color = Color::from_rgb8(0xa0, 0xa0, 0x9a);
 
+/// Added padding between each horizontal edge of the widget
+/// and the text in logical pixels.
```

Comment:
We should update the docs here to refer explicitly to `Label` rather than just "the widget".